web/client/epics/geostory.js

```diff
@@ -283,7 +284,19 @@ export const loadGeostoryEpic = (action$, {getState = () => {}}) => action$
                 return axios.get(`configs/${id}.json`)
                     // not return anything else that data in this case
                     // to match with data/resource object structure of getResource
-                    .then(({data}) => ({ data, isStatic: true, canEdit: true }));
+                    .then(({data}) => {
+                        // generates random id for section
+                        const defaultSection = data.sections[0];
```

Comment:
This section id replacement is specific for [newgeostory.json](https://github.com/geosolutions-it/MapStore2/blob/4ee11de7be140b78888fa304b6992a76e8feb5e2/web/client/configs/newgeostory.json) configuration but the string `id` could be different from `newgeostory` and it contain a diffferent structure of the configuration. could we add a control to add the uuid only if the `id` is equal to `newgeostory`?
What do you think @offtherailz ?